force-app/main/default/classes/SOQL.cls

```diff
@@ -143,15 +183,18 @@ public with sharing class SOQL {
      */
     public static List<String> getContactNamesByAccount(Id accountId) {
         // Create a list to hold the Contact LastNames
+        List<String> franksLastNameList = new List<String>();
 
         // Query the Contacts
+        List<Contact> franksContactList = [SELECT LastName FROM Contact WHERE AccountID = :accountId];
+
         // Loop through each Contact and add the LastName to the list
-        //for () {
-        //    
-        //}
+        for (Contact contact: franksContactList) {
+            franksLastNameList.add(contact.LastName);
+        }
```

Comment:
Good use case for a SOQL for loop. We don't really care to store the actual contacts, just the Last Names.